pkg/workload/workload.go

```diff
@@ -98,9 +99,18 @@ type AssignmentClusterQueueState struct {
 	ClusterQueueGeneration int64
 }
 
+// DRAInfoOptions holds DRA-specific configuration for workload.Info construction.
+// This keeps DRA concerns isolated from the main InfoOptions struct.
+type DRAInfoOptions struct {
+	enabled bool
```

Comment:
Do we need this? Why not just rely on the feature gate?

IIUC we could check DRA feature enablement based on the gate. If the DRA mapping is not supplied then we just have `preprocessedResources` empty.

Reply:
no this was left behind from previous commits, just pushed again removing it.